### ISSUE TITLE
The extension not working correctly anymore

### ISSUE THREAD

**Issue by dr-dark-flames** [Gerschel/sd-web-ui-quickcss#34]
I used to have the neon style but with different color themes
recently i just reinstalled the script and wanted to change the colors but its not even letting me?? there is no save new style there is no apply new colors and ontop of that the button themes and drop lists and the whole theme is not completey changed even when applying the default Neon style

**Comment by Nacurutu** [comment]
thank u for letting us know... as the last updates of the webUI brought too many issues, we r still waiting for a more stable commit to start looking for possible bugs.

**Comment by dr-dark-flames** [comment]
yeah I've noticed those updates always ruin everything T_T now my fav theme is gone T_T back to the default shit look..

**Comment by Nacurutu** [comment]
> yeah I've noticed those updates always ruin everything T_T now my fav theme is gone T_T back to the default shit look..

I recommend to use the last stable one..

git checkout a9fed7c

and after they fix the bugs, update.

**Comment by dr-dark-flames** [comment]
will do thank u!

**Comment by dr-dark-flames** [comment]
im sorry but how do i go back to an old version of auto1111? I tried doing that command but it kinda did nothing
```
Note: switching to 'a9fed7c'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at a9fed7c3 Merge pull request #8503 from mcmonkey4eva/filename-length-limit-fix
M       webui-user.bat
```

**Comment by dr-dark-flames** [comment]
and If i ever wanna update back to the new one how would i do that
